emulate.c

```diff
@@ -5,6 +5,7 @@
 
 #ifdef ENABLE_RV32F
 #include <math.h>
+#define RISCV_NAN 0x7fc00000
```

Comment:
Move the definition `RISCV_NAN` to header `riscv_private.h`, so that the incoming JIT compiler can make use of it as well.